PayItForward.Data.Services/Abstraction/IUsersService.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace PayItForward.Services.Data.Abstraction
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Dbmodel = PayItForward.Data.Models;
+
+    public interface IUsersService
+    {
+        // to be changed based on Controller needs
+        IQueryable<Dbmodel.User> GetAll();
+
+        Dbmodel.User GetById(string id);
+
+        Task<Dbmodel.User> GetByIdAsync(string id);
+
+        Dbmodel.User GetByUserName(string userName);
+
+        Task SaveAsync();
```

Comment:
You should remoce this method as it does not have any business logic, you will use save inside the other methods for example in AddUser service method